src/components/AddItemForm.js

```diff
@@ -28,22 +41,28 @@ class AddItemForm extends Component {
   }
 
   render () {
-    const { items, value } = this.state;
+    const { items, value } = this.state
+    const { classes } = this.props
     return (
-      <div>
+      <div className={classes.root}>
+      <AppBar position='static' color='default'>
         <label>
-          <input type="text" value={value} onChange={this.handleChange} className='addToDoInput' />
+          <TextField id='addToDoInput' multiline fullWidth label='Enter task here' value={value} onChange={this.handleChange} />
         </label>
-        <input
-          type='button'
-          value='Add To Do'
+        <span>
+        <Button
+            id='addToDoButton'
+            fullWidth
+            variant='contained'
           onClick={this.handleClick}
-          className='addToDoButton'
-        />
+          className={classes.button}
+        >Add task</Button>
+        </span>
+      </AppBar>
         {items.map(item => <p key={item} onClick={() => this.deleteItem(item)}>{item}</p>)}
       </div>
     )
   }
 }
 
-export default AddItemForm;
\ No newline at end of file
+export default withStyles(styles) (AddItemForm)
```

Comment:
convention is to leave no space between stuff like `withStyle(styles)(AddItemForm)`

Reply:
:+1: